implementation/src/main/java/io/smallrye/config/Expressions.java

```diff
@@ -0,0 +1,9 @@
+package io.smallrye.config;
+
+public class Expressions {
+    public static void withoutExpansion(final Runnable action) {
```

Comment:
@dmlloyd What this what you had in mind?

Reply:
More or less, yes.  It should be done in a `try`/`finally` though, and if expansion was already disabled it should not disable/enable it again.

```java
if (disable()) try { // returns true if the state was changed to disable
    action.run();
} finally {
    enable();
} else {
    action.run();
}
```

Reply:
Ok, pushed a new one. Check if it is better now. Thank you!